client/src/components/pages/common/Login/index.js

```diff
@@ -1,5 +1,125 @@
+import loginValidate from '../../../../utils/application/LoginValidation';
 
-const Login = () => <h1>Login</h1>;
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: null,
+      password: null,
+      role: null,
+      message: '',
+    };
+  }
+
+  handleError = () => {
+    window.location.href = '/404';
+  };
+
+  throwMessage = (message) => {
+    this.setState({ message });
+  };
+
+  login = () => {
+    this.setState({ message: '' });
+    const { email, password } = this.state;
+    loginValidate({
+      email,
+      password,
+    }).then((result) =>
+      result
+        ? axios
+            .post('/api/v1/login', {
+              email,
+              password,
+            })
+            .then((response) => {
+              this.setState({ role: response.data.role }, () => {
+                this.redirect();
+              });
+            })
+            .catch(() =>
+              this.setState({ message: 'Please Check your Email or Password' })
+            )
+        : this.setState({ message: 'Please Check your Email or Password' })
+    );
+  };
+
+  updateEmail = (e) => {
+    this.setState({ email: e.target.value });
+  };
+
+  updatePassword = (e) => {
+    this.setState({ password: e.target.value });
+  };
+
+  redirect() {
+    const { role } = this.state;
+    if (role === 'applicant') {
+      window.location.href = '/steps';
+    } else if (role === 'admin') {
+      window.location.href = '/dashboard';
+    }
+  }
+
+  render() {
+    const { message } = this.state;
+    return (
+      <ThemeProvider theme={Theme}>
+        <div className="container">
+          <SimpleCard
+            ClassName="card_sign"
+            content={
+              <>
+                <Typography variant="h4" align="center" color="primary">
+                  Sign In
+                </Typography>
+                <Typography variant="h6" align="center" color="secondary">
+                  Please enter your login details
+                </Typography>
+                <div className="login-input-container">
+                  <TextField
+                    id="email"
+                    label="Email"
+                    variant="outlined"
+                    type="email"
+                    placeholder="Enter Email"
+                    onChange={this.updateEmail}
+                  />
+                  <TextField
+                    id="password"
+                    label="Password"
+                    variant="outlined"
+                    type="password"
+                    placeholder="Enter Password"
+                    onChange={this.updatePassword}
+                  />
+                </div>
+                <Typography variant="body2" color="primary" align="left">
+                  {message}
+                </Typography>
+                <div className="login-button-container">
+                  <Button onClick={this.login}>Sign in</Button>
```

Comment:
we change it to Login to be similar to the route in the link, I think this is better
```suggestion
                  <Button onClick={this.login}>Login</Button>
```